node/pkg/governor/governor_db.go

```diff
@@ -89,6 +89,10 @@ func (gov *ChainGovernor) reloadPendingTransfer(pending *db.PendingTransfer, now
 
 	payload, err := vaa.DecodeTransferPayloadHdr(msg.Payload)
 	if err != nil {
+		if payload == nil {
```

Comment:
I think the real fix for this is to just delete lines 101 and 102 (don't reference payload on error).